dev/PushNotifications/PushNotificationsLongRunningTask.ProxyStub/NotificationsLongRunningProcess.idl

```diff
@@ -28,6 +28,10 @@ interface INotificationsLongRunningPlatform : IUnknown
     HRESULT RegisterForegroundActivator([in] IWpnForegroundSink* sink, [in] LPCWSTR processName);
 
     HRESULT UnregisterForegroundActivator([in] LPCWSTR processName);
+
+    HRESULT AddToastRegistration([in] LPCWSTR processName, [in] LPCWSTR toastGuid);
```

Comment:
If I understand the design correctly, this isn't always a GUID. It can also be an AUMID which is the appIdentifier for the packaged app. So this needs to be renamed appropriately. One idea is to just call it appId and leve it there.

Reply:
It can be GUID/AUMID. I will rename to appId.